nakadi/event_stream.py

```diff
@@ -7,72 +7,94 @@
 import sys
 
 
-def create_stream_generator(kafka_pool, topic, partition, start_from, opts):
+def create_stream_generator(kafka_pool, topic, cursors, opts):
+
+    topics = {}
+    for cursor in cursors:
+        topic_partition = (topic, int(cursor['partition']))
+        topics[topic_partition] = int(cursor['offset'])
```

Comment:
I would suggest an explicit check for the format of the partition and offset, though it could be ok for the hack